public/css/theme.css

```diff
@@ -1,3 +1,5 @@
+@import '/css/modules/waterpolo.css';
```

Comment:
Why are we now using module imports for the css over it being in one file? Is this supported on CasparCG / Chrome 63?

Reply:
Unless i'm mistaken, module imports have been a thing in CEF/Chrome for many years?

Reply:
But yes, consistency is nice.

Reply:
im planning on pulling the entire css into there own modules per sport, and having theme.css just input them all in order they are now

Reply:
and yeah its been around for a long time, plus I thought we was using the new beta release with CEF 3, so we can use the football field

Reply:
that way we have better seperation of concerns in the css world, its something, the whole thing is alot of mess now, needs a solid refactor :P

Reply:
I defo agree with the splitting up the modules, editing/adding of CSS has tended to make sports go out of order in the code.

Reply:
It's compounded by the fact I am struggling to scroll through the files because there's that much text (all be it I am using atom on this laptop :P ).
I will split them all out in a separate PR though, just this one as i didnt want to scroll no more.
All the html templates need this as well as the actually JS fiels, a move to more componentisation is ideal.

Reply:
For this one please add in a separate file - I am against a re-factor at the moment given all the other additions that are needed for the current roses and during a refactor there needs to be further design considerations given than lets just split it out into files.